rpc/server.py

```diff
@@ -400,11 +402,11 @@ async def call_contract_function(
         json.loads(function_call_data), logger=log_status
     )
 
-    return {
-        "status": "success",
-        "message": f"Function '{function_name}' called on contract at {contract_address} with args {args}.",
-        "execution_output": execution_output,
-    }
+    return_data['message'] = f"Function '{function_name}' called on contract at {contract_address} with args {args}."
+
+    return_data['data'] = execution_output
```

Comment:
```suggestion
    return_data['execution_output'] = execution_output
```
just to keep the format the same for now